### PR TITLE
At least expose the current single target in the CMS UI.

### PR DESCRIPTION
**Pull request by smn**
It'll introduce people to what's coming and makes the current hidden functionality more explicit.

### CODE REVIEW

**Comment by smn** [comment]
change_content_repository_information___django_site_admin

**Comment by smn** [comment]
@miltontony ready for review

**Comment by smn** [comment]
Has #29 merged in

File: cms/models.py [hunk 1]
```diff
@@ -79,36 +79,81 @@ class Meta:
...
+    @classmethod
+    def get_default_url(cls):
+        return settings.GIT_REPO_URL
+
+    name = models.CharField(
+        _('The name of the content repository'),
+        max_length=255, blank=True, null=True,
+        default=lambda: ContentRepository.get_default_name())
+    url = models.CharField(
+        _('Where Internet address of where this content repository lives'),
```
Comment (miltontony) [inline cms/models.py]: Typo. possibly `The Internet address of where..`

**Comment by miltontony** [comment]
@smn tests are failing..

**Comment by smn** [comment]
@miltontony thanks, fixed now I think.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**